src/components/Tiling.ts

```diff
@@ -77,12 +80,18 @@ export function tileBoard(world: WorldContainer) {
   }
 }
 
+export function isPointVisible(point: Point) {
+  return visibleTiles.some((c) => c.x === point.x && c.y === point.y);
```

Comment:
Would be good to use a consistent letter for these single-char values on simple iterators across the project, `e` in our case.